.secrets.baseline

```diff
@@ -141,5 +141,5 @@
       }
     ]
   },
-  "generated_at": "2021-12-10T00:03:20Z"
```

Comment:
Let’s be careful to stage changes via `git add -p` or similar review process so we don’t commit meaningless changes like this generated time.